config/locales/en.yml

```diff
@@ -715,6 +715,27 @@ en:
         - the client is financially independent
         - the client is in local authority or foster care
       guidance: Guidance on means aggregation for under 18s
+    how_to_aggregate:
+      title: You are going to aggregate your client's means with another person's means
+      panel_text: If your client's means are aggregated with the means of another person, such as a maintaining adult, they must have a full means test.
+      what_happens_next: What happens next
+      paragraphs_1:
+        - "For all questions that refer to 'your client' or 'the client', such as if your client has dependants or owns the home they live in, answer them with reference to both your client and the maintaining adult."
+        - "For questions that ask for financial information:"
+      bullets_1:
+        - add any relevant income or spending amounts for your client to those of the maintaining adult
+        - enter these combined amounts in the relevant fields that refer to 'your client'
+      paragraph_2_html: "If the maintaining adult has a partner, only include the partner if their finances can be included in the means assessment, as per regulation 16 of %{link}, and:"
```

Comment:
@patrick-laa unless I'm mistaken, but I don't think this link open's up into a new tab?

I think we have to change it to:

>     paragraph_2_html: "If the maintaining adult has a partner, only include the partner if their finances can be included in the means assessment, as per regulation 16 of <a href='%{link}' target='_blank'>The Civil Legal Aid (Financial Resources and Payment for Services) Regulations 2013 (opens in new tab)</a>, and:"

Then we can, get rid of` link_text:` and on `app/views/question_flow/how_to_aggregate.html.slim` update it to:

>   p.govuk-body
>     = t(".paragraph_2_html", link: document_link(:legislation_CLAR_2013_individual_resources))